src/directives/jf_tabs/jf_tabs.html

```diff
@@ -5,7 +5,7 @@
 	    ng-style="{width: jfTabs.getTabWidthForStyle()}"
 	    jf-disable-feature="{{ tab.feature }}">
 		<a ng-click="jfTabs.onClickTab(tab, true)" style="z-index: 999999">
-			<div class="jf-tab-header-container" jf-tooltip-on-overflow><span>{{jfTabs.dictionary[tab.name]}}</span>
+			<div class="jf-tab-header-container" jf-tooltip-on-overflow><span ng-bind-html="jfTabs.dictionary[tab.name]"></span>
```

Comment:
Following this change a tooltip will not appear when using HTML tags with overflowing text.
This means that the consuming code will have to add a tooltip on overflow.
In general I would rather have this change to be non-braking.
Please use an additional param, named `tabHeaderTemplate` and validate it's existence using `ng-if="tab.tabHeaderTemplate"` (make sure to shoe the html-binded div only if it exists and the other one else).